src/app/lib/config/services/afaanoromoo.js

```diff
@@ -0,0 +1,159 @@
+import { C_POSTBOX } from '@bbc/psammead-styles/colours';
+import { latin } from '@bbc/gel-foundations/scripts';
+import * as brandSVGs from '@bbc/psammead-assets/svgs';
+
+const service = {
+  lang: `om-ET`,
+  articleAuthor: `https://www.facebook.com/bbcnewsafaanoromoo`,
+  articleTimestampPrefix: 'Updated',
+  atiAnalyticsAppName: 'news-afaanoromoo',
+  brandName: 'BBC News Afaan Oromoo',
+  product: 'BBC News Afaan Oromoo',
+  defaultImage: 'https://news.files.bbci.co.uk/ws/img/logos/og/afaanoromoo.png',
+  defaultImageAltText: 'BBC News Afaan Oromoo',
+  dir: `ltr`,
+  externalLinkText: ', external',
+  imageCaptionOffscreenText: 'Image caption, ',
+  videoCaptionOffscreenText: 'Video caption, ',
+  audioCaptionOffscreenText: 'Audio caption',
+  defaultCaptionOffscreenText: 'Caption, ',
+  imageCopyrightOffscreenText: 'Image source, ',
+  locale: `om-ET`,
+  datetimeLocale: `om-ET`.toLowerCase(),
```

Comment:
This is used by moment, and expects the moment locale imported into this file. This will likely throw a console error as the locale youre requesting isnt provided to moment in simorgh.

Idm if you fix that in this PR or the next though